info.limpet/src/info/limpet/data/impl/LimpetList.java

```diff
@@ -0,0 +1,10 @@
+package info.limpet.data.impl;
+
+import java.util.ArrayList;
+
```

Comment:
@pecko  - could you put in a class level comment explaining why we're extending ArrayList?